website/pages-parts/CommonForms/ShippingForm.tsx

```diff
@@ -21,7 +21,6 @@ export function ShippingForm() {
       schema={schema}
       onSubmit={(model: any) => alert(JSON.stringify(model, null, 2))}
       model={model}
-      placeholder
```

Comment:
We'd like to add them in the schema then.

Reply:
I don't know what I am doing wrong, but the form behaves exactly like on [uniforms.tools](https://uniforms.tools/). The only empty fields `state` and `zip` receives the placeholder in the field level, so it seems like we don't need it.